test/CMakeLists.txt

```diff
@@ -46,11 +47,21 @@ foreach (algo_index RANGE 29)# 43
     if (algo_index STREQUAL 10)
       set (enable_ FALSE)
     endif ()
+    # Check if LUKSAN targets are available.
+    get_cmake_property(compile_defs_string COMPILE_DEFINITIONS)
+    string(FIND "${compile_defs_string}" "WITH_LUKSAN" has_luksan)
```

Comment:
just use NLOPT_WITH_LUKSAN

Reply:
Just looking for the option is not enough here, we need to extract and compare against actually used compile flags.

Reply:
reopening

Reply:
Done!